CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Latest Changes
+
+- Add support for empty batch dimension in `cuequivariance-torch`.
```

Comment:
nit-pick about what the changelog could look like
- other projects distinguish additions/fixes/etc. as in https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md for instance
- should we do the same?